org.eclipse.xtext/src/org/eclipse/xtext/documentation/impl/MultiLineJavaDocTypeReferenceProvider.java

```diff
@@ -110,4 +168,16 @@ protected void computeRegions(List<ReplaceRegion> regions, Iterable<ILeafNode> l
 			}
 		}
 	}
+
+	protected EReference getEReference(EObject eObject, INode node, int offset) {
```

Comment:
Please add JavaDoc to this method with a hint that this must be overridden. This should also be mentioned in the classes' JavaDoc.